docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -10,371 +9,131 @@ to save power and free CPU resources.
-   +----------------------------------+-------------------------+---------------+-----------------------------------------------------------------+
-   | ``PERF_COUNT``                   | ``YES``, ``NO``         | ``NO``        | Turns on performance counters reporting.                        |
-   +----------------------------------+-------------------------+---------------+-----------------------------------------------------------------+
-
-   The parameters are passed as strings to `IECore.load_network <api/ie_python_api/_autosummary/openvino.inference_engine.IECore.html#openvino.inference_engine.IECore.load_network>`_.
-
-   Normally, you do not need to select the execution target (``GNA_EXEC_TARGET``) and compilation target (``GNA_COMPILE_TARGET``). The default value for the execution target corresponds to available hardware, or latest hardware version supported by the plugin (i.e., GNA 3.0) if there is no GNA HW in the system. The compilation target is the same as the execution target by default. However, you may want to change the targets, for example, if you want to check how a model compiled for one generation would behave on the other generation (using the SW emulation mode), or if you are willing to export a model for a specific version of GNA HW.
-
-   You can change the ``GNA_DEVICE_MODE`` parameter at run time by sending a configuration dict to the `IECore.load_network <api/ie_python_api/_autosummary/openvino.inference_engine.IECore.html#openvino.inference_engine.IECore.load_network>`_ call, which works for any value excluding ``GNA_SW_FP32``. This enables you to switch the execution between software emulation mode and hardware execution mode after the model is loaded.
+Please refer to [Stateful models] (@ref openvino_docs_IE_DG_network_state_intro) for more details about such models.
 
-@endsphinxdirective
-## How to Interpret Performance Counters
+> **NOTE**: Typically, GNA is used in streaming scenarios, when minimizing the latency is important. Taking into account that POT does not support the `TensorIterator` operation, the recommendation is to use the `--transform` option of the Model Optimizer to apply `LowLatency2` transformation when converting an original model.
 
-With the following methods, you can collect performance counters that provides various performance data about execution on GNA:
+### Profiling
+The GNA plugin allows to turn on profiling using the `ov::enable_profiling` property.
+With the following methods, you can collect profiling information that provides various performance data about execution on GNA:
 
 @sphinxdirective
 .. tab:: C++
 
-   ``InferenceEngine::InferRequest::GetPerformanceCounts``
-
-   The returned map stores a counter description as a key, and a counter value in the ``realTime_uSec`` field of the ``InferenceEngineProfileInfo`` structure.
-
+   ``ov::InferRequest::get_profiling_info``
```

Comment:
do we need it to be code snippet? looks like it's a reference to a function, no more

Reply:
@ilya-lavrenov Currently profiling information is not aligned with other plugins, but we have plans to modify it in future and provide per-layer statistics. So I would like don't focus on it now.